pulsar-client/src/main/java/org/apache/pulsar/client/impl/BinaryProtoLookupService.java

```diff
@@ -131,7 +132,7 @@ public BinaryProtoLookupService(PulsarClientImpl client, String serviceUrl, bool
             }).exceptionally((sendException) -> {
                 // lookup failed
                 log.warn("[{}] failed to send lookup request : {}", destination.toString(), sendException.getMessage(),
-                        sendException);
+                        sendException instanceof ClosedChannelException ? null : sendException);
```

Comment:
instead null if we print empty string `""` then log will show empty string instead `{}`.

Reply:
The `{}` is there for the `sendException.getMessage()`, next arg is the exception object itself which doesn't need to have the `{}` placeholder.